README.md

```diff
@@ -57,3 +57,5 @@ steps:
 ```
 
 `cleanup` this defaults to true and will clean up the job and pod resources after the step runs. If set to false they will remain for 10 minutes to debug.
+
+`metadata` will add metadata (annotations) to the kubernetes job.
```

Comment:
Feels odd to put it inside metadata. Could we just use `annotations`?

Reply:
So I put it inside metadata because I think we are eventually going to need to add other things that live inside a [kubernetes object metadata](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#objectmeta-v1-meta). If we refactor it to be inside metadata later it'll be a breaking change. This was we can have other additions without breaking changes later. Kind of a coin flip TBH.